src/js/angular/guides/steps/complex/enable-autocomplete/plugin.js

```diff
@@ -0,0 +1,31 @@
+PluginRegistry.add('guide.step', [
+    {
+        'guideBlockName': 'enable-autocomplete',
+        'getSteps': (options, GuideUtils) => [
+            {
+                'guideBlockName': 'click-main-menu',
+                'options': angular.extend({}, {
+                    'label': 'menu_setup_autocomplete',
+                    'parentMenuSelector': 'menu-setup',
+                    'menuSelector': 'sub-menu-autocomplete'
+                }, options)
+            }, {
+                'guideBlockName': 'clickable-element',
+                'options': angular.extend({}, {
+                    'title': 'guide.step_plugin.enable-autocomplete.title',
+                    'content': 'guide.step_plugin.enable-autocomplete.content',
+                    'url': '/autocomplete',
+                    'elementSelector': GuideUtils.getGuideElementSelector('autocompleteCheckbox'),
+                    'onNextClick': () => {
+                        GuideUtils.waitFor(GuideUtils.getGuideElementSelector('autocompleteCheckbox', ' input'), 3, false)
+                            .then(autocompleteCheckbox => {
+                                if (!autocompleteCheckbox.is(':checked')) {
+                                    GuideUtils.clickOnGuideElement('autocompleteCheckbox')();
```

Comment:
So, this block actually enables the autocomplete by clicking the checkbox. I'm I right? Wasn't the guide's idea to just point the user what he/she should do and not to do it before himself?

Reply:
We talked about it, but we haven't made a final decision. We can remove the next button if we decide that we should continue after user interaction with the element. All clickable elements are designed to go on the next step by clicking on the element or click on next button.